undefined reference to lv_bidi_xxx

### LVGL version

v9.3.0-dev

### What happened?

after merge #6775 , I can't build it for esp32 port, the error log is 
<pre>
...

.espressif/tools/xtensa-esp32-elf/esp-2021r2-patch5-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: esp-idf/main/libmain.a(lv_mp.c.obj):(.rodata.mp_lv_bidi_process_paragraph_mpobj+0xc): undefined reference to `lv_bidi_process_paragraph'
.espressif/tools/xtensa-esp32-elf/esp-2021r2-patch5-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: esp-idf/main/libmain.a(lv_mp.c.obj):(.rodata.mp_lv_bidi_get_visual_pos_mpobj+0xc): undefined reference to `lv_bidi_get_visual_pos'
.espressif/tools/xtensa-esp32-elf/esp-2021r2-patch5-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: esp-idf/main/libmain.a(lv_mp.c.obj):(.rodata.mp_lv_bidi_get_logical_pos_mpobj+0xc): undefined reference to `lv_bidi_get_logical_pos'
.espressif/tools/xtensa-esp32-elf/esp-2021r2-patch5-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: esp-idf/main/libmain.a(lv_mp.c.obj):(.rodata.mp_lv_bidi_detect_base_dir_mpobj+0xc): undefined reference to `lv_bidi_detect_base_dir'
.espressif/tools/xtensa-esp32-elf/esp-2021r2-patch5-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld: esp-idf/main/libmain.a(lv_mp.c.obj):(.rodata.mp_lv_bidi_process_mpobj+0xc): undefined reference to `lv_bidi_process'

collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
ninja failed with exit code 1
make: *** [deploy] Error 2

</pre>

in my lv_conf.h, 

<pre>
...
#define LV_USE_PRIVATE_API		1
...
#define LV_USE_BIDI 0
</pre>

I think the macro LV_USE_BIDI judgement should be put in misc/lv_bidi_private.h too, after add it , I can build and flash firmware.

in lv_bidi_private.h

<pre>
/**********************
 * GLOBAL PROTOTYPES
 **********************/

#if LV_USE_BIDI

...

#endif
/**********************
 *      MACROS
 **********************/


</pre>











### How to reproduce?

_No response_

--- a/src/misc/lv_bidi_private.h
+++ b/src/misc/lv_bidi_private.h
@@ -15,6 +15,7 @@ extern "C" {
  *********************/
 
 #include "lv_bidi.h"
+#if LV_USE_BIDI
 
 /*********************
  *      DEFINES
@@ -90,6 +91,8 @@ void lv_bidi_process_paragraph(const char * str_in, char * str_out, uint32_t len
 /**********************
  *      MACROS
  **********************/
+
+#endif /*LV_USE_BIDI*/
 
 #ifdef __cplusplus
 } /*extern "C"*/

